src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import { FC } from "react";
+import { ManifestationsSimpleFieldsFragment } from "../../core/dbc-gateway/generated/graphql";
+import { useGetHoldingsV3 } from "../../core/fbs/fbs";
+import {
+  convertPostIdToFaustId,
+  creatorsToString,
+  filterCreators,
+  flattenCreators
+} from "../../core/utils/helpers/general";
+import Modal from "../../core/utils/modal";
+import { useText } from "../../core/utils/text";
+import { FaustId, Pid } from "../../core/utils/types/ids";
+import Disclosure from "../material/disclosures/disclosure";
+
+export const findOnShelfModalId = (faustId: FaustId) =>
+  `find-on-shelf-modal-${faustId}`;
+
+export interface FindOnShelfModalProps {
+  manifestation: ManifestationsSimpleFieldsFragment;
+}
+
+const FindOnShelfModal: FC<FindOnShelfModalProps> = ({ manifestation }) => {
+  const t = useText();
+  const {
+    pid,
+    creators,
+    titles: { main: mainTitle }
+  } = manifestation;
+  const faustId = convertPostIdToFaustId(pid as Pid);
+  const author =
+    creatorsToString(
+      flattenCreators(filterCreators(creators, ["Person"])),
+      t
+    ) || t("creatorsAreMissingText");
+  const { data, isError, isLoading } = useGetHoldingsV3({
+    recordid: [faustId]
+  });
+
+  if (isError || !data) {
+    // TODO: handle error once we have established a way to do it
+    return null;
+  }
+
+  const { holdings } = data[0];
+
+  return (
+    <Modal
+      modalId={findOnShelfModalId(faustId)}
+      screenReaderModalDescriptionText={t("screenReaderModalDescriptionText")}
+      closeModalAriaLabelText={t("ariaLabelModalTwoText")}
```

Comment:
Please rename these text entries.

The screen reader modal description (and close label) is intended to provide screen readers with a description (think headline) of a modal. We have multiple modals in a single app (as minimum: reservation + find on shelf) and they should not share description. This description should be related to finding items on a shelf. Since the manifestation title, author is a headline in the modal it should be OK not to include the title in the modal description.

Reply:
@Adamik10 You can take a look how Kasper named the same props in the reservation modal. We have introduced to prefix the keys with the context in order to prevent possible key collisions:
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/141/files#diff-bc5c522d9c285c7c50fe9142d7afdf0e1ddb84feb9cf5888bbb8735e5c162f50R77-R78

Reply:
Perfect comment Kasper! And thanks for inspiration Mikkel 😉